_overviews/scala3-migration/tooling-syntax-rewriting.md

```diff
@@ -157,37 +143,24 @@ case class State(n: Int, minValue: Int, maxValue: Int):
     do println(i + j)
 ```
 
-We moved to the new syntaxes of Scala 3 in two steps: first we used `-indent -rewrite` then `-new-syntax -rewrite`.
-We could also apply the new control structure syntax before the significant indentation syntax.
-But the compiler is not able to apply both at the same time: <del>`-indent -new-syntax -rewrite`</del>.
+We can observe that the number of lines was reduced by 2 thanks to the elimination of the closing curly braces.
```

Comment:
This reads as *divided by 2*, which is clearly not true. Consider removing that sentence. IMO it doesn't bring any value.